esmvalcore/preprocessor/_time.py

```diff
@@ -182,9 +178,52 @@ def get_time_weights(cube):
     return time_weights
 
 
-def daily_statistics(cube, operator='mean'):
+def hourly_statistics(cube, hours, operator='mean'):
+    """Compute hourly statistics.
+
+    Chunks time in x hours periods and computes statistics over them.
+
+    Parameters
+    ----------
+    cube: iris.cube.Cube
+        input cube.
+
+    hours: int
+        Number of hours per period. Must be a divisor of 24
+        (1, 2, 3, 4, 6, 8, 12, 24)
```

Comment:
This should probably not have the 24 then, since we only check up to 12.

```suggestion
        (1, 2, 3, 4, 6, 8, 12)
```